added wizard section to openEO web Editor documentation (#233)
eu-cdse/documentation: Applications/WebEditor.qmd

    As previously mentioned, it is necessary to log in to interact with the server. A new window will appear when attempting to log in, as demonstrated below. While other options are sometimes available, the recommended authentication choice is the "Copernicus Data Space Ecosystem". For further information regarding various authentication methods or to seek assistance, you can always click on the "help" option at the top or contact us.

>     <centre><img src="_images/webeditor/editor_login.PNG"  width="80%" height="60%"></centre>

Reviewer: note that this image is pretty outdated, it still contains "EGI" and "internal" options, which are dropped now

Author: updated the image

1. Load Collection 
   

>     a. To start a task within the openEO web editor, your first step involves ensuring you have the necessary dataset for the analysis. Thus, you must ensure the required dataset exists within the openEO database. You can do this by exploring/searching the list of available datasets from the sidebar of the interface.

Reviewer: use "collection" iso "dataset". Also "ensure" sounds a bit weird here

Author: corrected

>     b. In our example, we want to calculate the Normalized Difference Vegetation Index (NDVI), so the Sentinel 2 L2A collection will be used. Once you find this collection, the next step is to drag and drop it into the Process Editor for further operations.

Reviewer: I don't understand the difference between step a and b here, both are about finding the desired collection.

Author: combined and modified all the steps

        
    ![](_images/webeditor/drag_collection.png)

>     Now, once you load collection, there are several parameters you need to specify, including the area of interest, the temporal extent, and the selection of bands. When you click on the collection, a window pops up, where you can define the necessary parameters for subsequent processing. 

Reviewer: > When you click on the collection, a window pops up,

I think this should be clicking on any `load_collection` parameter, or clicking the "edit" icon in top right corner

Author: updated the sentence

>     c. To specify the area of interest, you can choose between generating a bounding box or importing your spatial extent by dragging and dropping GeoJSON or KML files onto the map view. Here, we've provided an example involving a bounding box. There is also an option to "No filter", that will include all the data in the datacube.

Reviewer 2: No bounding box filter is generally a bad idea for novice users (the ones that use a wizard). Instead we should point out that the user should start with a sufficiently small boundingbox with respect to the resolution of the data.

Author: updated the sentence

    ![](_images/webeditor/define_bbox.png)

>     d. Another parameter is Temporal extent, that imits the data to load from the collection to the specified  temporal interval.

Reviewer: ```suggestion
    d. Another parameter is temporal extent to limit the data to load to the specified time window.
```

    ![](_images/webeditor/define_temporal.png)

    e. In our case, for NDVI calculation, we have specifically chosen the Red band (B04) and the Near-Infrared (NIR) band (B08). 
        
    ![](_images/webeditor/define_bands.png)

2. Apply Processes
    

>     Eventually, the next step involves implementing essential processes, ranging from straightforward operations like adding bands to more complex tasks such as importing or defining user-defined functions (UDFs). 

Reviewer: ```suggestion
    The next step involves applying essential openEO processes, ranging from straightforward mathematical operations to more complex tasks such spatial or temporal aggregations. 
```

>     In this task, we're using a specific process called `reduce_dimension` as a key part of the operation. This process helps us simplify our collection: first, we use it to reduce temporal dimension and second, to deal with the various bands.

Reviewer: ```suggestion
    In this task, we'll using a specific openEO process called `reduce_dimension` two times to simplify our data cube. First to reduce the temporal dimension, and second to deal with the various bands.
```

            
    ![](_images/webeditor/reduce_dimension.png)

>     In the initial process, the `reduce_dimension` algorithm eliminates the temporal dimension by selecting the maximum value using the max function.

Reviewer: Note that, scientifically speaking, it is pretty weird to first do a temporal reduction of your raw sensor values and do NDVI calculation afterwards because you will be calculating the NDVI from red and nir values that might be unrelated. I think you first should do NDVI calculation and then taking the temporal max.

            
    <centre><img src="_images/webeditor/temporal_reducer.png"  width="80%" height="60%"></centre>

    Following this, the same `reduce_dimension` is utilised to reduce the band dimension after executing a series of addition, subtraction, and division operations necessary for the NDVI calculation.

    <centre><img src="_images/webeditor/reduce_bands.png"  width="80%" height="60%"></centre>

3. Select a format

    As a final step in the workflow creation is to select the output format. 
        
    ![](_images/webeditor/save_process.png)

>     Since our application involves simple NDVI calculation, we want to save it as a GeoTiff. 

Reviewer: ```suggestion
    Since our workflow eliminated the temporal dimension, we can keep things simple and just save the result as a GeoTiff. 
```
